cmd/controller-manager/app/controller_manager.go

```diff
@@ -303,7 +313,7 @@ func StartControllers(s *options.ControllerManagerServer,
 		glog.V(1).Info("Starting shared informers")
 		informerFactory.Start(stop)
 	} else {
-		glog.V(1).Infof("Skipping starting service-catalog controller because servicecatalog/v1alpha1 is not available")
+		return fmt.Errorf("unable to start service-catalog controller: servicecatalog/v1alpha1 is not available")
```

Comment:
See [this](https://kubernetes.io/docs/resources-reference/v1.6/#container-v1-core) for how it is configured in the Pod api.

This isn't a blocking issue for this PR, but does make things a bit nicer if we can do it easily.  It is a file that the termination message for a container is written to, and helps for quickly debugging issues.  e.g. `kubectl pod status` should print the termination message so you don't need to go trolling through logs.

Ideally we would do something like - configure glog.Fatal to write to both stderr + the termination path, and then always exit using glog.Fatal.